src/main/java/racingcar/domain/Cars.java

```diff
@@ -1,60 +1,70 @@
 package racingcar.domain;
 
-import racingcar.RandomNumberGenerator;
-
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
 import java.util.stream.Collectors;
 
 public class Cars {
-
     private static final int MAX_SIZE = 20;
+    private static final String NAME_DELIMITER = ",";
+
     private final List<Car> cars;
 
-    public Cars(final List<String> cars) {
+    public Cars(final List<Car> cars) {
         validateCars(cars);
-        validateSize(cars);
-        this.cars = cars.stream()
+        this.cars = cars;
+    }
+
+    public static Cars from (final List<Name> names) {
```

Comment:
1. 
```suggestion
    public static Cars from(final List<Name> names) {
```

2. 이 정적 팩터리 메서드는 `from(final List<Name> names)`인데, 아래 정적 팩터리 메서드는 `fromNameValues(final List<String> nameValues)`네요! 메서드명을 짓는 기준이 궁금하네요!

Reply:
사실 에러가 날 줄 모르고 둘 다 메서드명을  `from`으로 지었다가 에러가 발생해서 아래 메서드의 이름을 `fromNameValues`로 변경했습니다.

위에 메서드도 확실하게 `fromName`으로 바꾸는 게 좋을 것 같네요!